### PR TITLE
Add docs for advanced setting gui.sendBasicAuthPrompt (syncthing #8757)

### PR DESCRIPTION
**Pull request by emlun**
Companion PR of syncthing/syncthing#8757

### CODE REVIEW

**Comment by st-review** [comment]
:robot: beep boop

I'm going to close this pull request as it has been idle for more than 90 days.

This is not a rejection, merely a removal from the list of active pull requests that is periodically reviewed by humans. The pull request can be reopened when there is new activity, and merged once any remaining issues are resolved.

To permanently exempt this PR from bot nagging, add the `slow-pr` label.

**Comment by emlun** [comment]
The companion PR https://github.com/syncthing/syncthing/pull/8757 is nearing completion, can this be reopened? :slightly_smiling_face:

**Comment by acolomb** [comment]
Is this ready to merge now that the parent PR has landed?  Or do we have any last-minute changes that need docs updates?

**Comment by emlun** [comment]
As far as I'm aware, I believe this is ready to merge. Sorry for the delay!

File: users/config.rst [hunk 1]
```diff
@@ -899,6 +899,22 @@ The following child elements may be present:
     ``ldap``
         LDAP authentication. Requires ldap top level config section to be present.
 
+.. option:: gui.sendBasicAuthPrompt
+
+    The GUI previously used HTTP Basic Authorization for login, but now uses an
```
Comment (tomasz1986) [inline users/config.rst:904]: Just a quick comment, as I didn't have time to look at the PR before, but "previously" can mean literally anything. How about adding a specific version number here?

Comment (emlun) [reply users/config.rst]: Good catch! Agreed and fixed.

**Review by acolomb — APPROVED** [review]

**Pull request merged**